### PR TITLE
MAISTRA-2295 chart updates for ServiceImport resource support

### PR DESCRIPTION
**Pull request by rcernich**
Signed-off-by: rcernich <rcernich@redhat.com>

### CODE REVIEW

**Comment by rcernich** [comment]
This can be merged after maistra/istio#336 has been merged.

**Review by jwendell — APPROVED** [review]

File: resources/helm/v2.1/istio-init/files/serviceexports.maistra.io.crd.yaml [hunk 1]
```diff
@@ -142,20 +142,21 @@ spec:
...
                         properties:
-                          names:
+                          name:
                             type: string
                           namespace:
                             type: string
                         type: object
                       name:
                         properties:
-                          names:
```
Comment (dgn) [inline resources/helm/v2.1/istio-init/files/serviceexports.maistra.io.crd.yaml:152]: how come this changed? where do we get this from now, from maistra/api?

Comment (rcernich) [reply resources/helm/v2.1/istio-init/files/serviceexports.maistra.io.crd.yaml:152]: yes, but via maistra/istio

**Comment by rcernich** [comment]
Now that  maistra/istio#336 has been merged, installs will break until this is merged.

**Review by shamsher31 — APPROVED** [review]

**Pull request merged**